### ISSUE TITLE
Crash when logging event right after StartSession

### ISSUE THREAD

**Issue by alxbog** [mattleibow/Flurry.Analytics#8]
In my app I use this code on app launch:
```
FlurryAgent.StartSession("...");
FlurryAgent.LogEvent("MyFirstEvent");
```

The app crashes on the next main look.

```
Native stacktrace:

2015-05-28 12:31:12.034 MyAppiOS[3455:594946] critical: 	0   MyAppiOS                            0x000000010fbbcb4d mono_handle_native_sigsegv + 253
2015-05-28 12:31:12.034 MyAppiOS[3455:594946] critical: 	1   MyAppiOS                            0x000000010fbd0049 mono_sigsegv_signal_handler + 233
2015-05-28 12:31:12.034 MyAppiOS[3455:594946] critical: 	2   libsystem_platform.dylib            0x0000000118f1af1a _sigtramp + 26
2015-05-28 12:31:12.034 MyAppiOS[3455:594946] critical: 	3   ???                                 0x694c2f796578656c 0x0 + 7587491670652380524
2015-05-28 12:31:12.034 MyAppiOS[3455:594946] critical: 	4   MyAppiOS                            0x000000010fa8c7da +[FlurrySession sessionWithApiKey:startTime:] + 1171
2015-05-28 12:31:12.034 MyAppiOS[3455:594946] critical: 	5   MyAppiOS                            0x000000010fa9dd5e __29-[FlurryImpl setupForApiKey:]_block_invoke + 240
2015-05-28 12:31:12.034 MyAppiOS[3455
:594946] critical: 	6   libdispatch.dylib                   0x0000000118ba6f16 _dispatch_call_block_and_release + 12
2015-05-28 12:31:12.034 MyAppiOS[3455:594946] critical: 	7   libdispatch.dylib                   0x0000000118bc1964 _dispatch_client_callout + 8
2015-05-28 12:31:12.034 MyAppiOS[3455:594946] critical: 	8   libdispatch.dylib                   0x0000000118babcb4 _dispatch_queue_drain + 435
2015-05-28 12:31:12.034 MyAppiOS[3455:594946] critical: 	9   libdispatch.dylib                   0x0000000118bab9a3 _dispatch_queue_invoke + 217
2015-05-28 12:31:12.035 MyAppiOS[3455:594946] critical: 	10  libdispatch.dylib                   0x0000000118badc85 _dispatch_root_queue_drain + 534
2015-05-28 12:31:12.035 MyAppiOS[3455:594946] critical: 	11  libdispatch.dylib                   0x0000000118baed59 _dispatch_worker_thread3 + 98
2015-05-28 12:31:12.035 MyAppiOS[3455:594946] critical: 	12  libsystem_pthread.dylib             0x0000000118f2a637 _pthread_wqthread + 729
2015-05-28 12:31:12.035 MyAppiOS[3455:594946] critical: 	13  libsystem_pthread.dylib             0x0000000118f2840d start_wqthread + 13
2015-05-28 12:31:12.035 MyAppiOS[3455:594946] critical: 
=================================================================
Got a SIGSEGV while executing native code. This usually indicates
a fatal error in the mono runtime or one of the native libraries 
used by your application.
=================================================================
```

**Comment by alxbog** [comment]
Figured out that it's not because the event is timed.
It occurs because of logging event right after `StartSession`.

**Comment by alxbog** [comment]
Here is the description of the problem: http://stackoverflow.com/questions/30050917/flurry-crashing-in-ios

It's a bug in Flurry 6.3.0. It was fixed in 6.4.0.

@mattleibow 
Please update the lib to the latest version (6.5.0 by now). Thanks for great project! It really saves time for many developers.

**Comment by mattleibow** [comment]
Now on NuGet.org: https://www.nuget.org/packages/Flurry.Analytics/1.2.1
